### PR TITLE
feat: apply show more button at context-menu

### PR DESCRIPTION
**Pull request by skdud4659**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci`, minor refactoring, etc.)
- [ ] Need discussion
- [ ] Not that difficult
- [ ] Approved feature branch merge to master

### Description
SSIA
image

### CODE REVIEW

File: apps/web/src/services/dashboards/shared/dashboard-widget-input-form/dashboard-widget-options-form/DashboardCostWidgetValueOptionDropdown.vue [hunk 1]
```diff
@@ -74,17 +74,14 @@ const menuHandler: AutocompleteHandler = async (inputText, pageStart, pageLimit
...
-                page: {
-                    start: pageStart,
-                    limit: pageLimit,
-                },
             },
         };
         const { results } = await SpaceConnector.clientV2.costAnalysis.dataSource.list(param);
+        const refinedResultsItems = results[0][variableModelKeys[props.optionKey]].map((result) => (
+            { label: result, name: `${state.valueTarget}.${result}` }
+        ));
```
Comment (yuda110) [inline apps/web/src/services/dashboards/shared/dashboard-widget-input-form/dashboard-widget-options-form/DashboardCostWidgetValueOptionDropdown.vue:82]: nitpick: need defensive code for cases when `results` is empty!

**Review by yuda110 — APPROVED** [review]
LGTM!!!! I left a comment!

**Review by WANZARGEN — APPROVED** [review]
LGTM~

**Comment by WANZARGEN** [comment]
I love the size of this PR ㅎㅎ

**Pull request merged**